gpiozero/pins/mock.py

```diff
@@ -413,3 +385,34 @@ def tx_word(self, value, bits_per_word=None):
             bits = reversed(bits)
         self.tx_buf.extend(bits)
 
+
+class MockFactory(LocalPiFactory):
+    def __init__(self, revision='a21041', pin_class=MockPin):
```

Comment:
I wonder if these factory-parameters should also be settable through env-vars, like the pigpio factory params are? (to make testing from the command-line easier)